flow/peerdbenv/env.go

```diff
@@ -86,7 +87,15 @@ func decryptWithKMS(ctx context.Context, data []byte) ([]byte, error) {
 	return decrypted.Plaintext, nil
 }
 
+var KMSCache sync.Map
+
 func GetEnvBase64EncodedBytes(ctx context.Context, name string, defaultValue []byte) ([]byte, error) {
```

Comment:
this function is used for fetching Temporal cert and key too
are they also going to be KMS encrypted

also function name should indicate KMS dependency

Reply:
yes